src/ArcGISEarth.AutoAPI.Examples/packages.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.1.0" targetFramework="net461" />
+  <package id="Microsoft.Xaml.Behaviors.Wpf" version="1.1.19" targetFramework="net461" />
+  <package id="System.Buffers" version="4.5.1" targetFramework="net461" />
+  <package id="System.Memory" version="4.5.4" targetFramework="net461" />
+  <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="net461" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.7.1" targetFramework="net461" />
+  <package id="System.Text.Encodings.Web" version="4.7.1" targetFramework="net461" />
+  <package id="System.Text.Json" version="4.7.2" targetFramework="net461" />
+  <package id="System.Threading.Tasks.Extensions" version="4.5.4" targetFramework="net461" />
+  <package id="System.ValueTuple" version="4.5.0" targetFramework="net461" />
```

Comment:
@yong7743  Can new packages be added ?

Reply:
Are they necessary? And I think it should be from the system, not on the Nuget. Even on Nuget we should use package reference instead of the packages.config.